### PR TITLE
Added examples for generated AuditEvents

### PR DESCRIPTION
**Pull request by eugene-blokhin**

### CODE REVIEW

File: _static/files/audit-event-examples/R4_read.json [hunk 1]
```diff
@@ -0,0 +1,122 @@
+{
+    "resourceType": "AuditEvent",
+    "type": {
+        "system": "http://hl7.org/fhir/audit-event-type",
```
Comment (alexzautke) [inline _static/files/audit-event-examples/R4_read.json:4]: System changed in R4, needs to be "http://terminology.hl7.org/CodeSystem/audit-event-type"

File: _static/files/audit-event-examples/R4_read.json [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        "code": "rest",
+        "display": "RESTful Operation"
+    },
+    "subtype": [
+        {
+            "system": "http://hl7.org/fhir/http-verb",
```
Comment (alexzautke) [inline _static/files/audit-event-examples/R4_read.json:10]: We could add a display value to both codings

File: _static/files/audit-event-examples/R4_read.json [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+                ]
+            },
+            "requestor": true,
+            "network": {
+                "address": "127.0.0.1",
+                "type": "2"
+            }
+        },
+        {
+            "type": {
```
Comment (alexzautke) [inline _static/files/audit-event-examples/R4_read.json:44]: An empty agent with no content is not useful. It's the client right? I would merge it with the other agent or move the network property down here. Also add another "requestor": "true" to it.

File: _static/files/audit-event-examples/R4_read.json [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+        "type": [
+            {
+                "system": "http://hl7.org/fhir/security-source-type",
+                "code": "4",
+                "display": "Application Server"
+            }
+        ],
+        "observer": {
+            "identifier": {
+                "system": "http://vonk.fire.ly/fhir/sid/devices",
```
Comment (alexzautke) [inline _static/files/audit-event-examples/R4_read.json:66]: We should remove the "vonk" part here

File: _static/files/audit-event-examples/R4_read.json [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+        }
+    ],
+    "action": "R",
+    "recorded": "2022-10-07T14:21:15.456+00:00",
+    "outcome": "0",
+    "outcomeDesc": "200",
+    "agent": [
+        {
+            "altId": "unknown",
+            "who": {
```
Comment (alexzautke) [inline _static/files/audit-event-examples/R4_read.json:25]: We should make sure that we adhere to https://profiles.ihe.net/ITI/BALP/content.html#35754-oauth-mapping-to-auditevent and map the iss and sub fields to agent.who correctly

File: _static/files/audit-event-examples/R4_read.json [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+    "entity": [
+        {
+            "what": {
+                "identifier": {
+                    "value": "REQUEST_ID"
+                },
+                "type": "Device"
+            },
+            "type": {
+                "system": "http://fire.ly/fhir/sid/api-diagnostics",
```
Comment (alexzautke) [inline _static/files/audit-event-examples/R4_read.json:81]: Here we should use http://terminology.hl7.org/CodeSystem/audit-entity-type as it's an extensible binding in the IHE IG

File: _static/files/audit-event-examples/R4_read.json [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+        },
+        {
+            "what": {
+                "identifier": {
+                    "value": "CONNECTION_ID"
+                },
+                "type": "Device"
+            },
+            "type": {
+                "system": "http://fire.ly/fhir/sid/api-diagnostics",
```
Comment (alexzautke) [inline _static/files/audit-event-examples/R4_read.json:93]: Same as above

File: _static/files/audit-event-examples/R4_read.json [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+                "code": "ConnectionId"
+            }
+        },
+        {
```
Comment (alexzautke) [inline _static/files/audit-event-examples/R4_read.json:97]: We should use also "System object" as the type here

**Review by alexzautke — CHANGES_REQUESTED** [review]

File: features/auditing.rst [hunk 9]
```diff
@@ -183,4 +183,16 @@ There is no further configuration for AuditEvent logging. If you include it in t
 
 For transactions and batches the audit plugin will create an AuditEvent for the transaction/batch as a whole *and* one for every entry in the transaction/batch.
 
-Firely Server does not allow you to update or delete the AuditEvent resources through the RESTful API so the Audit log cannot be tampered with. You can of course still manipulate these resources directly on the database, for instance to offload a surplus of old AuditEvent resources elsewhere. Please :ref:`vonk-contact` us for details if you want to do this.
\ No newline at end of file
+Firely Server does not allow you to update or delete the AuditEvent resources through the RESTful API so the Audit log cannot be tampered with. You can of course still manipulate these resources directly on the database, for instance to offload a surplus of old AuditEvent resources elsewhere. Please :ref:`vonk-contact` us for details if you want to do this.
+
+You can download examples of AuditEvent resources generated by the server below:
```
Comment (alexzautke) [inline features/auditing.rst:188]: Downloading the audit events is a good start. For readability I would suggest to create a table comparing the structured log elements with the FHIRPath of the element in the resource.

**Review by alexzautke — CHANGES_REQUESTED** [review]

**Review by alexzautke — CHANGES_REQUESTED** [review]
Some audit events don't validate, can you check that?

**Comment by alexzautke** [comment]
That would also be a nice addition for the Postman collection, first retrieve the audit event resource and then pipe it into $validate and make sure that everything is OK

File: _static/files/audit-event-examples/R4_read.json (hunk 2 above)
Comment (eugene-blokhin) [reply _static/files/audit-event-examples/R4_read.json:10]: Do I just re-use the values from the `code` element?

File: _static/files/audit-event-examples/R4_read.json (hunk 3 above)
Comment (eugene-blokhin) [reply _static/files/audit-event-examples/R4_read.json:44]: I want to discuss it with you in a call to make sure I understand you

File: _static/files/audit-event-examples/R4_read.json [hunk 10]
```diff
@@ -0,0 +1,122 @@
...
+            "code": "read"
+        }
+    ],
+    "action": "R",
+    "recorded": "2022-10-07T14:21:15.456+00:00",
+    "outcome": "0",
+    "outcomeDesc": "200",
+    "agent": [
+        {
+            "altId": "unknown",
```
Comment (eugene-blokhin) [inline _static/files/audit-event-examples/R4_read.json:24]: Remove

File: _static/files/audit-event-examples/R4_read.json [hunk 11]
```diff
@@ -0,0 +1,122 @@
...
+            "system": "http://hl7.org/fhir/restful-interaction",
+            "code": "read"
+        }
+    ],
+    "action": "R",
+    "recorded": "2022-10-07T14:21:15.456+00:00",
+    "outcome": "0",
+    "outcomeDesc": "200",
+    "agent": [
+        {
```
Comment (eugene-blokhin) [inline _static/files/audit-event-examples/R4_read.json:23]: Add `"policy": "jtt"`

File: _static/files/audit-event-examples/R4_read.json (hunk 6 above)
Comment (eugene-blokhin) [reply _static/files/audit-event-examples/R4_read.json:81]: Don't show this information is Auth isn't used

File: _static/files/audit-event-examples/R4_read.json [hunk 12]
```diff
@@ -0,0 +1,122 @@
...
+                "coding": [
+                    {
+                        "system": "https://profiles.ihe.net/ITI/BALP/CodeSystem/UserAgentTypes",
+                        "code": "UserOauthAgent",
+                        "display": "User OAuth Agent participant"
+                    }
+                ]
+            },
+            "requestor": true,
+            "network": {
```
Comment (eugene-blokhin) [inline _static/files/audit-event-examples/R4_read.json:38]: move to `agent:client`

File: _static/files/audit-event-examples/R4_read.json (hunk 6 above)
Comment (eugene-blokhin) [reply _static/files/audit-event-examples/R4_read.json:81]: Let's discuss it whenever you have time

**Review by alexzautke — APPROVED** [review]

**Pull request merged**